### DISCUSSION TITLE
Project Translation

### DISCUSSION THREAD

**Discussion by shuvashish76** [gbtami/pychess-variants#860] [Q&A] [answered]
I've seen OSS projects use these sites for translation purpose
- [Crowdin](https://crowdin.com/)
- [Transifex](https://www.transifex.com/)
- [Weblate](https://weblate.org/en/)

I think Weblate is better as it's [opensource](https://github.com/WeblateOrg/weblate).

Features: https://weblate.org/en/features/
![Screenshot_2021-12-05-21-15-09](https://user-images.githubusercontent.com/48226312/144753433-4fc61e84-f12d-42d0-885e-1b9abd399061.jpeg)
https://weblate.org/en/hosting/

---

(_Lichess itself use Crowdin : https://crowdin.com/project/lichess_)

I don't see any such project translation link on pychess website [about](https://www.pychess.org/about) section or on GitHub README.md

Though I remember there was a pychess discord support channel for Translators (but well I don't use discord anymore)

Please add Translation documentation/instructions on README.md.

ThankYou🙏🏿

**Comment by gbtami** [comment, marked as answer]
We use transifex. See https://github.com/gbtami/pychess-variants/wiki/Translations

**Reply by comradekingu** [reply]
But there is no community there, and it is a horrendous tool.
Compare this to https://hosted.weblate.org/projects/dreamchess/#languages

**Comment by shuvashish76** [comment]
Platform choice discussion for lichess: https://github.com/lichess-org/lila/issues/13740

**Reply by gbtami** [reply]
Telling the truth I maintain the Hungarian translations myself and never have any issues using transifex.
